metricbeat/metricbeat.yml

```diff
@@ -46,7 +46,13 @@ metricbeat.modules:
   period: 10s
   processes: ['.*']
 
-
+#------------------------------- Jolokia Module ------------------------------
```

Comment:
If you use `short_config: false`in the module fields.yml it will not show up here. See https://github.com/elastic/beats/blob/master/metricbeat/module/kafka/_meta/fields.yml#L7